.github/workflows/release.yaml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on:
+  push:
+    branches: ["main"]
+    paths:
```

Comment:
How do `paths` work?

Reply:
The action will trigger only upon changes in the files listed in paths

Reply:
Oh, thanks! I remember I started looking into it but then forgot. We should use them in the other repos as well